### DISCUSSION TITLE
Frankenphp for apps served under directory

### DISCUSSION THREAD

**Discussion by PrasadChinwal** [php/frankenphp#698] [Q&A] [answered]
We serve our `Laravel` apps under a directory for ex. https://apps.xyz.com/application_1. 

I have installed and configured Frankenphp with Laravel and the issue is that my url changes to https://apps.xyz.com/application_1/index.php 

I tried this fix in caddy file but didn't help:
```
php_server {
    index frankenphp-worker.php
    # Required for the public/storage/ directory...
    resolve_root_symlink
    # Required to serve index.html using only the folder
    try_files {path} {path}/index.html {path}/index.php frankenphp-worker.php
}
```
I still see `/index.php` in the url.
Can someone help me with this?

**Comment by withinboredom** [comment]
I think it is this line:

```
try_files {path} {path}/index.html {path}/index.php frankenphp-worker.php
```

if there isn't a file, it will fall back to index.html, index.php, and then frankenphp-worker.php; in that order. If you have an index.php, it will redirect to index.php.

**Reply by PrasadChinwal** [reply]
Thanks for the reply! So in order to configure it to route to laravel what changes would be required. 
Note that i can still see my laravel app landing page. It's just the issue with url which appends index.php.

**Reply by withinboredom** [reply, marked as answer]
Note the docs for laravel don't include that:

https://frankenphp.dev/docs/laravel/

You can learn more about what is going on under the hood here:

https://frankenphp.dev/docs/config/

(hint: `try_files` isn't a valid `php_server` directive, so it seems you are experiencing undefined behavior)
